### PR TITLE
Output image instead of intials if one is provided

### PR DESCRIPTION
**Pull request by adampetrie**

### CODE REVIEW

**Comment by dileshni** [comment]
Looks good to me :dog:

**Comment by alexBaizeau** [comment]
:shipit:

**Comment by f3ndot** [comment]
:shipit:

File: addon/components/initials-avatar.js [hunk 1]
```diff
@@ -3,9 +3,14 @@ export default Ember.Component.extend({
   firstName: '',
   lastName: '',
   company: '',
+  image: '',
   colorIndex: 1,
   maxColorIndex: 1,
 
+  hasImage: function() {
+    return Ember.isPresent(this.get('image'));
+  }.property('image'),
```
Comment (calvinlough) [inline addon/components/initials-avatar.js]: Would Ember.computed.notEmpty work here?

**Comment by skaasten** [comment]
:+1:

**Pull request merged**